sdk/containerregistry/container-registry/samples-dev/setImageProperties.ts

```diff
@@ -0,0 +1,35 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+/**
+ * @summary Illustrates how to update the properties on the tag so it can't be overwritten or deleted.
+ * @azsdk-weight 5
+ */
+
+// This sample assumes the registry has a repository `hello-world` with image tagged `v1`.
+
+import { ContainerRegistryClient } from "@azure/container-registry";
+import { DefaultAzureCredential } from "@azure/identity";
+import * as dotenv from "dotenv";
+dotenv.config();
+
+async function main() {
+  // Get the service endpoint from the environment
+  const endpoint = process.env.CONTAINER_REGISTRY_ENDPOINT || "<endpoint>";
+  // Create a new ContainerRegistryClient
+  const client = new ContainerRegistryClient(endpoint, new DefaultAzureCredential());
+  const image = client.getArtifact("library/hello-world", "latest");
```

Comment:
is `client.getArtifact` awaitable?

Reply:
no, this is a function to create an instance of the client type `RegistryArtifact`.  No service call is made.